src/Engine/IO/Window.cpp

```diff
@@ -117,14 +150,28 @@ void Window::update(float dt) {
     Mouse::update(x, y);
 
     /* Update ImGui */
+    // TODO: clean up this code
+    static bool imGuiActive = false;
```

Comment:
Should be static vars be defined in the header?

Reply:
it's pertinent only to the function.

Reply:
realized this isn't necessary and removed it